zookeeper-server/src/main/java/org/apache/zookeeper/ZooDefs.java

```diff
@@ -32,6 +32,14 @@
 
     public static final String ZOOKEEPER_NODE_SUBTREE = "/zookeeper/";
 
+    /**
+     * WARN: please don't retain the order, which is used to check 
```

Comment:
did you mean "don't change" or "retain", as opposed to "don't retain"? my understanding is the order must be preserved here for the code to work.